aspnetcore/tutorials/min-web-api.md

```diff
@@ -898,29 +946,32 @@ A DTO may be used to:
 
 To demonstrate the DTO approach, update the `Todo` class to include a secret field:
 
-[!code-csharp[](min-web-api/samples/6.x/todoDTO/Program.cs?name=snippet_secret&highlight=6)]
+:::code language="csharp" source="min-web-api/samples/7.x/todoDTO/Program.cs" id="snippet_secret" highlight="6":::
 
 The secret field needs to be hidden from this app, but an administrative app could choose to expose it.
 
 Verify you can post and get the secret field.
 
 Create a DTO model:
 
-[!code-csharp[](min-web-api/samples/6.x/todoDTO/Program.cs?name=snippet_DTO)]
+:::code language="csharp" source="min-web-api/samples/7.x/todoDTO/Program.cs" id="snippet_DTO":::
 
 Update the code to use `TodoItemDTO`:
 
+:::code language="csharp" source="min-web-api/samples/7.x/todoDTO/Program.cs" id="snippet_all":::
 [!code-csharp[](min-web-api/samples/6.x/todoDTO/Program.cs?name=snippet_all)]
 
-Verify you can't post or get the secret field.
+Verify you can post and get all fields except the secret field.
 
-<a name="diff-v6"></a>
+<a name="diff-v7"></a>
 
 ## Differences between minimal APIs and APIs with controllers
 
-- No support for filters: For example, no support for  <xref:Microsoft.AspNetCore.Mvc.Filters.IAsyncAuthorizationFilter>, <xref:Microsoft.AspNetCore.Mvc.Filters.IAsyncActionFilter>, <xref:Microsoft.AspNetCore.Mvc.Filters.IAsyncExceptionFilter>, <xref:Microsoft.AspNetCore.Mvc.Filters.IAsyncResultFilter>,  and <xref:Microsoft.AspNetCore.Mvc.Filters.IAsyncResourceFilter>.
-- No support for model binding, i.e. <xref:Microsoft.AspNetCore.Mvc.ModelBinding.IModelBinderProvider>, <xref:Microsoft.AspNetCore.Mvc.ModelBinding.IModelBinder>. Support can be added with a custom binding shim.
-  - No support for binding from forms. This includes binding <xref:Microsoft.AspNetCore.Http.IFormFile>. We plan to add support for `IFormFile` in the future.
+Minimal APIs have:
+
+- Different support for filters. For more information, see [Filters in Minimal API apps](../fundamentals/minimal-apis/min-api-filters.md)
+- No support for model binding, for example: <xref:Microsoft.AspNetCore.Mvc.ModelBinding.IModelBinderProvider>, <xref:Microsoft.AspNetCore.Mvc.ModelBinding.IModelBinder>. Support can be added with a custom binding shim.
+  - No support for binding from forms, except for <xref:Microsoft.AspNetCore.Http.IFormFile>. For more information, see [File uploads using IFormFile and IFormFileCollection](xref:aspnetcore-7#file-uploads-using-iformfile-and-iformfilecollection).
 - No built-in support for validation, i.e. <xref:Microsoft.AspNetCore.Mvc.ModelBinding.Validation.IModelValidator>
 - No support for [application parts](xref:mvc/extensibility/app-parts) or the [application model](xref:mvc/controllers/application-model). There's no way to apply or build your own conventions.
 - No built-in view rendering support. We recommend using [Razor Pages](xref:tutorials/razor-pages/razor-pages-start) for rendering views.
```

Comment:
Under this is No support for [ApiVersioning](https://www.nuget.org/packages/Microsoft.AspNetCore.Mvc.Versioning/). See [this issue](https://github.com/dotnet/aspnet-api-versioning/issues/751) for more details.

that should be removed